charts/firefly/templates/_helpers.tpl

```diff
@@ -245,4 +261,13 @@ tokens:
     name: erc1155
     url: http://{{ include "firefly.fullname" . }}-erc1155.{{ .Release.Namespace }}.svc:{{ .Values.erc1155.service.port }}
 {{- end }}
+{{- if and .Values.config.tokensOverride (not .Values.erc20.enabled) }}
+tokens:
+    {{- tpl .Values.config.tokensOverride . | nindent 2 }}
+{{- else if .Values.erc20.enabled }}
```

Comment:
Everything in this PR looks great! Only thing that might need correct is this bit. Since `tokens` is already getting configured for `erc1155` we'll want to just do a sort of `or` to append `erc20` config to the `tokens` plugin list if its set. So lines 255 - 272 should be replaced with something like:
```yaml
# realized we had a "bug" here, a user should be able to use `config.tokensOverride` even if either `erc1155` or `erc20` is enabled
{{- if and .Values.config.tokensOverride }}
tokens:
    {{- tpl .Values.config.tokensOverride . | nindent 2 }}
{{- else if or .Values.erc1155.enabled .Values.erc20.enabled }}
tokens:
  {{- if .Values.erc1155.enabled }}
  - plugin: fftokens
    name: erc1155
    url: http://{{ include "firefly.fullname" . }}-erc1155.{{ .Release.Namespace }}.svc:{{ .Values.erc1155.service.port }}
  {{- end }}
  {{- if .Values.erc20.enabled }}
    - plugin: fftokens
    name: erc20
    url: http://{{ include "firefly.fullname" . }}-erc20.{{ .Release.Namespace }}.svc:{{ .Values.erc20.service.port }}
  {{- end }}
{{- end }}
```